server/id/id.go

```diff
@@ -45,14 +46,29 @@ type allocatorImpl struct {
 	base uint64
 	end  uint64
 
-	client   *clientv3.Client
-	rootPath string
-	member   string
+	client    *clientv3.Client
+	rootPath  string
+	allocPath string
+	label     string
+	member    string
+	metrics   *metrics
+}
+
+// metrics is a collection of idAllocator's metrics
+type metrics struct {
```

Comment:
Why we need to have this type instead of using prometheus.Guage directly?

Reply:
It was to improve the readability and make it easier to add more metrics, as suggested https://github.com/tikv/pd/pull/5284#discussion_r919728599

Reply:
Once we recover PD, do we also need to recover all id allocators?